services/configurationService.js

```diff
@@ -9,6 +13,33 @@ class ConfigurationService {
         }, {"$limit": 1}]);
         return (result?.length === 1) ? result[0] : null;
     }
+
+    /**
+     * API: getPBTCDDefaults retrieve roles default permissions and notifications.
+     * @param {*} params 
+     * @param {*} context 
+     */
+    async getPBACDefaults(params, context){
+        verifySession(context)
+            .verifyInitialized()
+            .verifyRole([ROLES.ADMIN]);
+
+        return await this.getPBACByRoles(params.roles);
+    }
+    /**
+     * Get PBAC defaults by roles
+     * @param {Array} roles
+     * @returns {Object} PBAC defaults
+     */
+    async getPBACByRoles(roles){
```

Comment:
Can you set it private #getPBACByRoles?

Reply:
It's not private, other service will use this function.